exporters/otlp/otlplog/otlploggrpc/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	go.opentelemetry.io/otel/log v0.12.0
 	go.opentelemetry.io/otel/sdk v1.36.0
 	go.opentelemetry.io/otel/sdk/log v0.12.0
-	go.opentelemetry.io/otel/sdk/log/logtest v0.12.0
+	go.opentelemetry.io/otel/sdk/log/logtest v0.0.0-20250521073539-a85ae98dcedc
```

Comment:
But when we use this [tool](https://github.com/open-telemetry/opentelemetry-go/blob/a85ae98dcedc0761078518a715dea53e519b4846/Makefile#L312), it will most likely revert to v0.12.0.

I'm considering whether I need to move the logic of testing this part to the `tests` directory and handle it as ignored in [versions.yaml](https://github.com/open-telemetry/opentelemetry-go/blob/a85ae98dcedc0761078518a715dea53e519b4846/versions.yaml#L41).

Reply:
I think this is a bug in multimod. It shouldn't upgrade to undefined versions.

Reply:
We can consider also making releases of `logtest` modules...